dali/command.py

```diff
@@ -64,7 +64,8 @@ def value(self):
         if self._value is None and self._expected:
             raise MissingResponse()
         if self._value and self._value.error and not self._error_acceptable:
-            raise ResponseError()
+            print(ResponseError())
```

Comment:
Debug ?

Reply:
same here. I think it is not ok to crash the software if something goes wrong